src/libs/SearchUIUtils.ts

```diff
@@ -1564,11 +1671,6 @@ function createTypeMenuSections(
         const isEligibleForUnapprovedCompanyCardsSuggestion = isAdmin && isApprovalEnabled && cardFeedsForDisplayPerPolicy[policy.id]?.length > 0;
         const isEligibleForReconciliationSuggestion = false; // s77rt TODO
 
-        // The default feed must be based on an eligible policy
-        if ((isEligibleForStatementsSuggestion || isEligibleForUnapprovedCompanyCardsSuggestion) && !defaultFeed) {
-            defaultFeed = cardFeedsForDisplayPerPolicy[policy.id]?.sort((a, b) => a.name.localeCompare(b.name)).at(0)?.id;
-        }
```

Comment:
There is now a redundant `activePolicy` added to the iteration array, remove it with the comment "The active (default) policy is prioritized..."